extcrypto/extcrypto.cc

```diff
@@ -6,12 +6,19 @@
 #include <openssl/pem.h>
 #include <openssl/bio.h>
 #include <openssl/bn.h>
+#include <nan.h>
 
 namespace extcrypto {
 
+  using Nan::Utf8String;
```

Comment:
nit: Move this to the bottom of the block to keep it alphabetically ordered.

Reply:
Good catch! updated.